tests/test_interface.nim

```diff
@@ -1,6 +1,6 @@
 ## TODO: Split these up
 ## Maybe see if I can store a base state of the process before each tests runs and roll back after
-import osproc, streams, os
+import osproc, streams, os, sequtils
```

Comment:
`sequtils` isn't used, did you intend to?

Reply:
I thought it might be required for `in`, but I guess not. Removed now

Reply:
nice, I saw the warning when running the tests :ok_hand: